Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -50,14 +49,14 @@
 REQUEST_STATUS_PENDING = 'Pending'
 
 # Configuration Details
-username = credentials.username
-password = credentials.password
```

Comment:
This was right in the first place!
We do want to keep credentials file but just not push it into github.
We want to delete configuration file though